### PR TITLE
waterfall: prevent tooltip from flashing in Firefox

### PR DESCRIPTION
**Pull request by rodrigc**
Cherry-picked from https://github.com/buildbot/buildbot/pull/3296 by @Lekensteyn

### CODE REVIEW

**Comment by Lekensteyn** [comment]
Thanks, does 0cfa6c2bd6e351a3c479e54661ba84649e6ef2d1 also need to be applied? Last time I checked the tooltip showed on the wrong side of the cursor because of the wrong order

**Comment by rodrigc** [comment]
@Lekensteyn I'm not sure about that.  What would be helpful is if you could submit only https://github.com/buildbot/buildbot/commit/0cfa6c2bd6e351a3c479e54661ba84649e6ef2d1 on a new branch in a new PR, and paste a before and after screenshot showing what it fixes.
For this UI kind of stuff, it is hard to understand the impact without screenshots.

File: www/waterfall_view/src/module/main.module.coffee [hunk 1]
```diff
@@ -470,6 +470,7 @@ class Waterfall extends Controller
             else "150,0 170,#{height / 2} 150,#{height} 0,#{height} 0,0"
         tooltip = e.append('g')
             .attr('class', 'svg-tooltip')
+            .style('pointer-events', 'none')
```
Comment (tardyp) [inline www/waterfall_view/src/module/main.module.coffee]: I'd rather put this style in styles.less
```css
.svg-tooltip {
pointer-events: 'none';
}
```

Comment (rodrigc) [reply www/waterfall_view/src/module/main.module.coffee]: OK, I changed it

File: www/waterfall_view/src/styles/styles.less [hunk 2]
```diff
@@ -219,3 +219,7 @@
     }
   }
 }
+
+.svg-tooltip {
+  pointer-events: 'none';
```
Comment (Lekensteyn) [inline www/waterfall_view/src/styles/styles.less]: Any reason why this is not just:

    pointer-events: none;

https://developer.mozilla.org/en-US/docs/Web/CSS/pointer-events

Comment (rodrigc) [reply www/waterfall_view/src/styles/styles.less]: @Lekensteyn I changed the syntax to match the format on the developer.mozilla.org web site

**Pull request merged**